### PR TITLE
Optimize build

### PR DESCRIPTION
**Pull request by christianalfoni**
Also has a package.json fix on "none"

### CODE REVIEW

**Comment by Guria** [comment]
Have comments here, but tomorow

File: scaffold/snabbdom/webpack.config.js [hunk 1]
```diff
@@ -12,14 +37,11 @@ module.exports = {
     loaders: [{
       test: /\.js?$/,
       exclude: /node_modules/,
-      loader: 'babel-loader',
+      loader: 'babel',
       query: {
-        "presets": ["es2015", "stage-0"],
-        "plugins": []
+        presets: ['react', 'es2015', 'stage-0']
```
Comment (bfitch) [inline scaffold/snabbdom/webpack.config.js]: Does snabbdom need these?

File: scaffold/none/webpack.config.js [hunk 2]
```diff
@@ -1,18 +1,47 @@
...
   },
+  devServer: {
+    port: 3000
+  },
   module: {
     loaders: [{
       test: /\.js?$/,
       exclude: /node_modules/,
-      loader: 'babel-loader',
+      loader: 'babel',
```
Comment (bfitch) [inline scaffold/none/webpack.config.js:29]: I never know which is correct 'babel' or 'babel-loader'. What's the difference?

**Comment by bfitch** [comment]
This is great! I know nothing about webpack optimization, so thanks! :)

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cerebral-cli",
-  "version": "0.0.13",
+  "version": "0.0.15",
```
Comment (Guria) [inline package.json]: 0.0.15 is current version in master/ seems you forgot to use latest master when you start branch

File: scaffold/inferno/webpack.config.js [hunk 4]
```diff
@@ -1,8 +1,33 @@
+'use strict';
+
+const webpack = require('webpack');
+const path = require('path');
+let plugins = [];
```
Comment (Guria) [inline scaffold/inferno/webpack.config.js]: what about?

``` js
const plugins = [
new webpack.DefinePlugin({
    'process.env': {
      'NODE_ENV': JSON.stringify(process.env.NODE_ENV)
    }
  })
];
if (process.env.NODE_ENV === 'production') {
  plugins.push(new webpack.optimize.UglifyJsPlugin({ compress: { warnings: false } }))
}
```

File: scaffold/inferno/webpack.config.js [hunk 5]
```diff
@@ -12,11 +37,11 @@ module.exports = {
     loaders: [{
       test: /\.js?$/,
       exclude: /node_modules/,
-      loader: 'babel-loader',
+      loader: 'babel',
       query: {
-        "presets": ["es2015", "stage-0"],
-        "plugins": ["inferno"]
+        presets: ['react', 'es2015', 'stage-0']
```
Comment (Guria) [inline scaffold/inferno/webpack.config.js]: wrong config for snabbdom

File: scaffold/none/webpack.config.js (hunk 2 above)
Comment (Guria) [inline scaffold/none/webpack.config.js:29]: @bfitch 
no difference. both work.
https://webpack.github.io/docs/using-loaders.html#resolving-loaders
https://webpack.github.io/docs/configuration.html#resolveloader-moduletemplates

File: scaffold/none/webpack.config.js [hunk 6]
```diff
@@ -1,18 +1,47 @@
...
       query: {
-        "presets": ["es2015", "stage-0"]
+        presets: ['react', 'es2015', 'stage-0']
```
Comment (Guria) [inline scaffold/none/webpack.config.js]: react not needed here

File: scaffold/snabbdom/webpack.config.js (hunk 1 above)
Comment (Guria) [inline scaffold/snabbdom/webpack.config.js]: @bfitch 
I think it is copy paste error. inferno and none also has wrong config

File: scaffold/inferno/webpack.config.js (hunk 4 above)
Comment (christianalfoni) [inline scaffold/inferno/webpack.config.js]: Cool, looks better!

**Comment by Guria** [comment]
@christianalfoni bump it to 0.0.16

**Comment by christianalfoni** [comment]
Done :)

**Comment by christianalfoni** [comment]
hmf, conflict wtf?

**Comment by Guria** [comment]
@christianalfoni try to pull master and rebase this branch on master

**Comment by christianalfoni** [comment]
I do not think I have this scenario in my git notes :) How do I do that?

**Comment by Guria** [comment]
git checkout master
git pull --rebase
git checkout optimizeBuild
git rebase master

File: package.json [hunk 7]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "cerebral-cli",
   "version": "0.0.15",
+  "version": "0.0.16",
```
Comment (Guria) [inline package.json:3]: one field is extra :)

**Comment by Guria** [comment]
@bfitch looks ready for merge and release

**Pull request merged**